conf/application.conf

```diff
@@ -218,6 +218,12 @@ microservice {
       host = "localhost"
       port = 9498
     }
+
+    transit-movements-converter {
```

Comment:
As a note: check to see if this is in app-config-base, if not, it's worth doing it now.